### DISCUSSION TITLE
在GPU模式下运行PaddleNLP报错OSError: (External) CUBLAS error(7)

### DISCUSSION THREAD

**Discussion by zhijianli** [PaddlePaddle/PaddleNLP#2452] [General]
ubuntu 20.04
CUDA 11.2　
cudnn 8.1.1
paddlepaddle-gpu 版本：2.3.0.post112
python 3.8
安装完成之后测试：

import paddle
paddle.utils.run_check()

报错:
OSError: (External) CUBLAS error(7). 
  [Hint: 'CUBLAS_STATUS_INVALID_VALUE'.  An unsupported value or parameter was passed to the function (a negative vector size, for example). To correct: ensure that all the parameters being passed have valid values. ] (at /paddle/paddle/phi/backends/gpu/gpu_context.cc:424)

![image](https://user-images.githubusercontent.com/33283007/172571102-0fcdcbd8-cc7a-4c12-9d14-c287c034b5db.png)
求教是什么原因？

**Comment by ZeyuChen** [comment]
@wawltor

**Reply by wawltor** [reply]
建议Conda一下安装飞桨， https://www.paddlepaddle.org.cn/
